coalib/parsing/StringProcessing/Core.py

```diff
@@ -328,6 +328,39 @@ def escape(string, escape_chars, escape_with="\\"):
     return string
 
 
+def convert_to_raw(string, exceptions=""):
+    """
+    Converts a string to its raw form, converting all backslash to double
+    backslash except when the backslash escapes a character given in
+    exceptions.
+
+    :param string:     The given string that needs to be converted
+    :param exceptions: A list of characters that if escaped with backslash
+                       should not be converted to double backslash.
+    :return:           Returns the corresponding raw string.
+    """
+    always_exceptions = ['\'', '\"', 'a', 'b', 'f', 'n', 'r', 't', 'v', '/']
```

Comment:
https://docs.python.org/2.0/ref/strings.html

These are the characters that are interpreted differently in Python. So I think we should maintain that consistency here as well.